bridge/client/app/_views/ktb-settings-view/ktb-integration-view/ktb-integration-view.component.ts

```diff
@@ -1,158 +1,102 @@
-import { Component, OnDestroy, OnInit, TemplateRef } from '@angular/core';
+import { Component, TemplateRef } from '@angular/core';
 import { DtSortEvent, DtTableDataSource } from '@dynatrace/barista-components/table';
-import { BehaviorSubject, combineLatest, Observable, Subject } from 'rxjs';
+import { combineLatestWith, merge, Observable, shareReplay, Subject, switchMap } from 'rxjs';
 import { DataService } from '../../../_services/data.service';
 import { ActivatedRoute, Router } from '@angular/router';
-import { filter, map, switchMap, takeUntil } from 'rxjs/operators';
+import { distinctUntilChanged, filter, finalize, map, tap } from 'rxjs/operators';
 import { UniformRegistrationLog } from '../../../../../shared/interfaces/uniform-registration-log';
 import { UniformRegistration } from '../../../_models/uniform-registration';
 import { Location } from '@angular/common';
-import { UniformSubscription } from '../../../_models/uniform-subscription';
+
+export type Params = { projectName: string; integrationId?: string };
+
+const sortConfig: Record<string, (u: UniformRegistration) => string> = {
+  host: (u) => u.metadata.hostname,
+  namespace: (u) => u.metadata.kubernetesmetadata.namespace,
+  location: (u) => u.metadata.location,
+};
+
+const empty = '--EMPTY--';
 
 @Component({
   selector: 'ktb-keptn-services-list',
   templateUrl: './ktb-integration-view.component.html',
   styleUrls: ['./ktb-integration-view.component.scss'],
 })
-export class KtbIntegrationViewComponent implements OnInit, OnDestroy {
-  private readonly unsubscribe$ = new Subject<void>();
-  private selectedUniformRegistrationId$ = new Subject<string>();
-  private uniformRegistrationLogsSubject = new BehaviorSubject<UniformRegistrationLog[]>([]);
-  public UniformRegistrationClass = UniformRegistration;
+export class KtbIntegrationViewComponent {
+  private selectUniformRegistrationId$ = new Subject<string>();
   public isLoadingUniformRegistrations = true;
-  public uniformRegistrations: DtTableDataSource<UniformRegistration> = new DtTableDataSource();
-  public selectedUniformRegistration?: UniformRegistration;
-  public uniformRegistrationLogs$: Observable<UniformRegistrationLog[]> =
-    this.uniformRegistrationLogsSubject.asObservable();
   public isLoadingLogs = false;
-  public projectName?: string;
   public lastSeen?: Date;
+  private uniformRegistrations: DtTableDataSource<UniformRegistration> = new DtTableDataSource();
+
+  public params$: Observable<Params> = this.route.paramMap.pipe(
+    filter((paramMap) => !!paramMap.get('projectName')),
+    map((paramMap) => ({
+      projectName: paramMap.get('projectName') ?? empty,
+      integrationId: paramMap.get('integrationId') ?? undefined,
+    }))
+  );
+
+  private registrations$ = this.dataService
+    .getUniformRegistrations()
+    .pipe(finalize(() => (this.isLoadingUniformRegistrations = false)));
```

Comment:
Is currently executed twice, when accessing the page

Reply:
Fixed